devops/kubernetes/charts/origin/templates/relayer.deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "relayer.fullname" . }}
+  labels:
+    app: {{ template "relayer.fullname" . }}
+    app.kubernetes.io/name: origin
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: frontend
+    app.kubernetes.io/part-of: origin-relayer
+spec:
+  replicas: {{ default 1 .Values.relayerReplicas }}
+  selector:
+    matchLabels:
+      app: {{ template "relayer.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "relayer.fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+      - name: origin-relayer
+        image: "{{ .Values.containerRegistry }}/{{ .Release.Namespace }}/{{ .Values.relayerImage }}:{{ .Values.relayerImageTag }}"
+        imagePullPolicy: Always
+        env:
+        - name: ENVKEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "relayer.fullname" . }}
+              key: ENVKEY
+        ports:
+        - containerPort: 4001
+      - name: cloudsql-proxy
```

Comment:
If there is no database access this container should be removed

Reply:
For future DB needs...